pages/search.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div>
+    <v-app-bar fixed flat dense class="second-toolbar">
+      <v-tabs v-model="searchTab" centered>
+        <v-tab :key="0">Top results</v-tab>
+        <v-tab :key="1">Movies</v-tab>
+        <v-tab :key="2">Shows</v-tab>
+        <v-tab :key="3">Albums</v-tab>
+        <v-tab :key="4">Tracks</v-tab>
+        <v-tab :key="5">Books</v-tab>
+        <v-tab :key="6">People</v-tab>
+        <v-tab :key="7">Artists</v-tab>
```

Comment:
I think we can put those and the actual tabs in a v-for no?

Reply:
They need translating too.

These we could probably do. The pages, the idea is to be able to easily use a component inside of a page if we want later on.